compiler_opt/rl/inlining/inlining_runner.py

```diff
@@ -71,6 +71,9 @@ def compile_fn(
       cancelled work.
       RuntimeError: if llvm-size produces unexpected output.
     """
+    if cancellation_manager is None:
```

Comment:
why even pass it as an arg, why not we remove it from the arglist and use self._cancellation_manager?

Reply:
changed it, prob should check with yundi@ to see if that's an ok thing to do